src/title.rs

```diff
@@ -13,31 +13,31 @@ use crate::{layout::Alignment, text::Line};
 ///
 /// Title with no style.
 /// ```
-/// # use ratatui::widgets::block::Title;
+/// use ratatui::widgets::block::Title;
+///
 /// Title::from("Title");
 /// ```
 ///
 /// Blue title on a white background (via [`Stylize`](crate::style::Stylize) trait).
 /// ```
-/// # use ratatui::widgets::block::Title;
-/// # use ratatui::style::Stylize;
+/// use ratatui::{prelude::*, widgets::block::Title};
+///
 /// Title::from("Title".blue().on_white());
 /// ```
 ///
 /// Title with multiple styles (see [`Line`] and [`Stylize`](crate::style::Stylize)).
 /// ```
-/// # use ratatui::widgets::block::Title;
-/// # use ratatui::style::Stylize;
-/// # use ratatui::text::Line;
+/// use ratatui::{prelude::*, widgets::block::Title};
+///
 /// Title::from(
 ///     Line::from(vec!["Q".white().underlined(), "uit".gray()])
 /// );
 /// ```
 ///
 /// Complete example
 /// ```
-/// # use ratatui::widgets::block::{Title, Position};
-/// # use ratatui::layout::Alignment;
+/// use ratatui::{prelude::*, widgets::{*, block::*}};
```

Comment:
`Title` is fully qualified in the above two changes but implicitly imported here?

Reply:
Good catch - I've fixed this in a few places just now.
Title is a weird one in terms of where it's defined (only affects block, but is defined outside of block and is a little too generic a name to put in the prelude nicely.